indico/modules/rb/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -47,6 +47,17 @@ class BookingEditForm extends React.Component {
     hideOptions: {},
   };
 
+  showWarning = (initialValues, values) => {
+    return !(
+      moment(values['timeSlot']['startTime'], 'HH:mm').isSameOrAfter(
+        moment(initialValues['timeSlot']['startTime'], 'HH:mm')
+      ) &&
```

Comment:
`||`, remove the negation and use the negated versions of isSameOrAFter / isSameOrBefore